lib/oneboxer.rb

```diff
@@ -80,6 +84,33 @@ def self.invalidate(url)
     Discourse.cache.delete(onebox_failed_cache_key(url))
   end
 
+  def self.cache_response_body?(uri)
+    uri = URI.parse(uri) if uri.is_a?(String)
+
+    if SiteSetting.cache_onebox_response_body?
+      SiteSetting.cache_onebox_response_body_domains.split("|").map { |domain| uri.hostname.ends_with?(domain) }.any?
+    end
+  end
+
+  def self.cache_response_body(uri, response)
+    key = redis_cached_response_body_key(uri)
+    Discourse.redis.without_namespace.setex(key, 1.minutes.to_i, response).present?
```

Comment:
Do we need `.present?`? Is the return value ever used?

```suggestion
    Discourse.redis.without_namespace.setex(key, 1.minutes.to_i, response)
```